tests/flow/tests_pytorch.py

```diff
@@ -481,19 +481,19 @@ def test_pytorch_scriptrun_errors(env):
         env.assertEqual(type(exception), redis.exceptions.ResponseError)
         env.assertEqual("WRONGTYPE Operation against a key holding the wrong kind of value", exception.__str__())
 
-    # ERR no script at key from SCRIPTRUN
+    # ERR no script at key from SCRIPTEXECUTE
```

Comment:
You can use the `check_error_message` util (from includes.py) to save a lot of replicated code here...